### PR TITLE
Support actionpack 5.2

### PR DESCRIPTION
**Pull request by Edouard-chin**
- Move all Gemfile inside a `gemfiles` directory
- Test the gem with actionpack 5.2
- Added the Gemfile.lock under version Control (see commit for the explanation)

@pi3r @bdewater 

cc/ @rafaelfranca

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,176 @@
+PATH
```
Comment (rafaelfranca) [inline Gemfile.lock]: Don't forget to remove the Gemfile.lock in the CI

Comment (Edouard-chin) [reply Gemfile.lock]: Do we need to in this case? Since CI doesn't even run with the regular Gemfile. Only the one inside the `gemfiles` folder

Comment (rafaelfranca) [reply Gemfile.lock]: Ah, so we don't need

**Comment by bdewater** [comment]
Thanks!

**Pull request merged**